### ISSUE TITLE
Direct moving window to master area 

### ISSUE THREAD

**Issue by lluke777** [fuhsjr00/bug.n#162]
Let say we have 4 windows:
1 - master on left side.
2, 3 and 4 are on the right side in one column.

Is there a shortcut or way to directly move  window nr. 3 to master area?
Can it be done with one shortcut? For ex: Alt+3 ?

**Comment by joten** [comment]
<kbd>Win</kbd><kbd>Shift</kbd><kbd>Enter</kbd> does that, if window no. 3 is the active one. `View_shuffleWindow(1)` moves the active window to the first position in the view's window list, which is in the master area, unless it is already there, in which case it will be moved to the second position.

**Comment by lluke777** [comment]
Thank you, for your answer. What I have in mind is that the main window(1) is active and I move window nr. 3 with shortcut let say Alt + 3 to  master area.  Is there such a function?
Is there a function in *.ahk files  that  sets focus on given window ?

**Comment by joten** [comment]
No, currently there is no such function, neither for moving nor activating the nth window. There are similar functions and it would be possible to implement the functionality in question.
